### PR TITLE
Update plugin-loader.php docs

### PR DESCRIPTION
**Pull request by emrikol**
The instructions for wpcom_vip_load_plugin are a big ambiguous if you want to load client-mu-plugins.

### CODE REVIEW

File: client-mu-plugins/plugin-loader.php [hunk 1]
```diff
@@ -11,5 +11,10 @@
...
 
 // wpcom_vip_load_plugin( 'plugin-name' );
-// Note the above requires a specific naming structure: /plugin-name/plugin-name.php
-// You can also specify a specific root file: wpcom_vip_load_plugin( 'plugin-name/plugin.php' );
+/**
+ * Note the above requires a specific naming structure: /plugin-name/plugin-name.php
+ * You can also specify a specific root file: wpcom_vip_load_plugin( 'plugin-name/plugin.php' );
+ *
+ * wpcom_vip_load_plugin only loads plugins from the `WP_PLUGIN_DIR` directory.
+ * For client-mu-plugins `requre __DIR__ '/plugin-name/plugin-name.php'` works.
```
Comment (jenkoian) [inline client-mu-plugins/plugin-loader.php]:
```suggestion
 * For client-mu-plugins `require __DIR__ '/plugin-name/plugin-name.php'` works.
```

Comment (emrikol) [reply client-mu-plugins/plugin-loader.php]: @jenkoian I think we both missed the `.` ;)

`require __DIR__ . '/plugin-name/plugin-name.php'`

**Comment by emrikol** [comment]
@scottsweb Still happy with this after the recent change? :D

**Comment by scottsweb** [comment]
Yup, good catch.

**Pull request merged**